src/components/organisms/Sidebar.tsx

```diff
@@ -23,7 +23,7 @@ function Sidebar({
 
   /* FIXME  */
   return (
-    <div css={sidebarStyle(isMobile)}>
+    <div css={sidebarStyle(isMobile, !router.query.sectionId)}>
```

Comment:
```suggestion
    <div css={sidebarStyle(isMobile, router.query.sectionId == null)}>
```

요렇게 명확하게 표현해줘도 좋을 것 같슴다!

Reply:
오..! 혹시 `===`이 아니라 `==`을 제안하신 이유가 있으신건가용?? (궁금)

Reply:
undefind 도 같이 잡으려고...?!

Reply:
https://github.com/Lubycon/career-zip-frontend/pull/9#discussion_r637549007
앗ㅋㅋㅋ 다른 리뷰에서 이미 이유를 말씀해주셨네요
감사함니당 건우님🙇‍♂️